gruntfile.js

```diff
@@ -1,30 +1,27 @@
-let _ = require('underscore');
-
 module.exports = function (grunt) {
 
-  grunt.loadNpmTasks('grunt-contrib');
-  grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-concat');
-
+  // Project
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
-    concat: {
```

Comment:
This is not right. You shouldn't delete this because we need it to compile the JS files. the sass related task should not be appended to this file and not this file be replaced all together.

Reply:
Grunt wasn't installed as a dev Dependency before I added in Sass, so it appeared as though that Gruntfile wasn't doing anything. How were the tasks being run if Grunt was not installed as a package?